variables.tf

```diff
@@ -27,21 +22,52 @@ variable "model_name" {
   default     = ""
```

Comment:
This former default behaviour would lead to s3 buckets being called "-model" etc. This is the reason for the introduction of `resource_naming_prefix`